openshift/ups-template.json

```diff
@@ -0,0 +1,529 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "redhat-mobile-backend",
+    "annotations": {
+      "description": "An example Mobile Backend using RedHat, JBoss, AeroGear, and FeedHenry technologies",
+      "tags": "mobile",
+      "iconClass": "icon-node"
+    }
+  },
+  "parameters": [
+    {
+      "name": "NAMESPACE",
+      "displayName": "Namespace",
+      "description": "The OpenShift Namespace where the ImageStream resides.",
+      "value": "openshift"
+    }
+  ],
+  "objects": [
+    {
+      "kind": "Secret",
+      "apiVersion": "v1",
+      "metadata": {
+        "name": "unifiedpush"
+      },
+      "stringData": {
+        "database-user": "unifiedpush",
+        "database-password": "unifiedpush",
```

Comment:
I don't think I will be able to extract these to parameters without updating the UPS image.  Right now these values are hard coded into the liquibase migration scripts.

@matzew WDYT?